app/helpers/publishers_helper.rb

```diff
@@ -33,64 +40,64 @@ def publisher_humanize_balance(publisher, currency)
     I18n.t("helpers.publisher.balance_unavailable")
   end
 
-  def next_deposit_date(today = DateTime.now)
-    if today.day > 8
-      today = today + 1.month
+  def publisher_converted_overall_balance(publisher)
+    return if publisher.default_currency == "BAT" || publisher.default_currency.blank?
+    overall_balance = publisher.wallet&.overall_balance
+    if overall_balance&.amount_default_currency.present?
+      I18n.t("helpers.publisher.balance_pending_approximate",
+             amount: '%.2f' % overall_balance.amount_default_currency,
+             code: overall_balance.default_currency)
+    else
+      I18n.t("helpers.publisher.conversion_unavailable", code: publisher.default_currency)
     end
-    today.strftime("%B 8th")
   end
 
-  def publisher_converted_balance(publisher)
-    currency = publisher.default_currency
-    return if currency == "BAT" || currency.blank?
-    if balance = publisher.wallet &&
-        publisher.wallet.contribution_balance.is_a?(Eyeshade::Balance) &&
-        publisher.wallet.contribution_balance
-      converted_amount = '%.2f' % balance.convert_to(currency)
-      I18n.t("helpers.publisher.balance_pending_approximate", amount: converted_amount, code: currency)
+  def publisher_channel_bat_balance(publisher, channel_identifier)
+    channel_balance = publisher.wallet&.channel_balances.dig(channel_identifier)
+    if channel_balance&.amount_bat.present?
+      '%.2f' % balance.amount_bat
```

Comment:
Whoops, this is my fault: `balance` here should be `channel_balance`, or else channel balances will display to the publishers as 'Unavailable'.

![image](https://user-images.githubusercontent.com/12549658/53036739-5d594f80-3446-11e9-8c4e-9b70ee1d895e.png)